onnx/defs/traditionalml/defs.cc

```diff
@@ -18,7 +18,7 @@ ONNX_ML_OPERATOR_SET_SCHEMA(
         .Input(
             1,
             "Y",
-            "The index values to select as a int64 tensor",
+            "The indices, which must be at least one and no more than the number of dimensions of 'X'.",
```

Comment:
no more than number of dimensions of 'X' is misleading. number of dimensions means "rank" of 'X".

The meaning here is, no more than the size of 'X'.